doc/rst/dcp.1.rst

```diff
@@ -11,26 +11,37 @@ DESCRIPTION
 
 Parallel MPI application to recursively copy files and directories.
 
-dcp is a file copy tool in the spirit of :manpage:`cp(1)` that evenly distributes
-work across a large cluster without any centralized state. It is
+dcp is a file copy tool in the spirit of :manpage:`cp(1)` that evenly
+distributes the work of scanning the directory tree, and copying file
+data across a large cluster without any centralized state.  It is
 designed for copying files that are located on a distributed parallel
-file system.
+file system, and will split large file copies across multiple threads.
```

Comment:
"across multiple processes."  So far, we don't have any threads.  Threads likely would improve performance for us, at least for small process counts, but it'll take some effort to add.